software/TOOL/SUBTOOL/meta.yml

```diff
@@ -1,25 +1,24 @@
 ## TODO nf-core: Please delete all of these TODO statements once the file has been curated
-## TODO nf-core: Change the name of "software_tool" below
-name: software_tool
+## TODO nf-core: Change the name of "tool_subtool" below
+name: tool_subtool
 ## TODO nf-core: Add a description and keywords
-description: Run FastQC on sequenced reads
+description: Sort SAM/BAM/CRAM file
 keywords:
-    - quality control
-    - qc
-    - adapters
-    - fastq
+    - sort
+    - bam
+    - sam
+    - cram
 tools:
-    ## TODO nf-core: Change the name of "software" below
-    - software:
+    ## TODO nf-core: Change the name of the tool below
+    - samtools:
         ## TODO nf-core: Add a description and other details for the software below
         description: |
-            FastQC gives general quality metrics about your reads.
-            It provides information about the quality score distribution
-            across your reads, the per base sequence content (%A/C/G/T).
-            You get information about adapter contamination and other
-            overrepresented sequences.
-        homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
-        documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
+            SAMtools is a set of utilities for interacting with and post-processing
+            short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+            These files are generated as output by short read aligners like BWA.
+        homepage: http://www.htslib.org/
+        documentation: hhttp://www.htslib.org/doc/samtools.html
+        doi: 10.1093/bioinformatics/btp352
```

Comment:
Personally I think we can omit the `homepage` because most tools don't have any, and if the do, it's the documentation anyways (as is the case here, too). I think `documentation` and `doi` are really helpful though.

Reply:
With documentation I always go with the attitude that the more the better. I think we can just be lenient as to how we `lint` for it instead?

Reply:
Yeah that's a good point :)